src/shared/internal-calendar/calendar-manager.tsx

```diff
@@ -355,7 +355,7 @@ const Component = (
 
     const renderHeader = () => {
         return (
-            <Header data-id="calendar-header">
+            <Header data-testid="calendar-header">
```

Comment:
`data-id` is used to target certain child elements when styling with styled-components. example is `time-slot-week-view` which has custom padding for the header

if `data-testid` is still needed, can set it as an additional prop instead